tools/topology/topology2/cavs-rt5682.conf

```diff
@@ -262,6 +297,18 @@ Object.PCM {
 			formats	'S16_LE,S24_LE,S32_LE'
 		}
 	}
+
+	pcm."27" {
+		name	"echo-reference"
+		id	27
+		direction	"capture"
+		Object.Base.fe_dai."echo-reference" {}
```

Comment:
can we please wait till https://github.com/thesofproject/sof/pull/6986 merged and update this to use instance ID to instantiate?

Reply:
Sure